setup.py

```diff
@@ -35,14 +37,91 @@ def is_conda_interpreter():
     """
     return 'conda' in sys.version or 'Continuum' in sys.version
 
+def is_macOS():
+    return 'darwin' in sys.platform
+
+
+# HOMEBREW_SAMPLE = """$ brew ls --verbose libomp
+# /opt/homebrew/Cellar/libomp/15.0.7/INSTALL_RECEIPT.json
+# /opt/homebrew/Cellar/libomp/15.0.7/.brew/libomp.rb
+# /opt/homebrew/Cellar/libomp/15.0.7/include/ompt.h
+# /opt/homebrew/Cellar/libomp/15.0.7/include/omp.h
+# /opt/homebrew/Cellar/libomp/15.0.7/include/omp-tools.h
+# /opt/homebrew/Cellar/libomp/15.0.7/lib/libomp.dylib
+# /opt/homebrew/Cellar/libomp/15.0.7/lib/libomp.a
+# """
+#
+# MACPORTS_SAMPLE = """$ port contents libomp
+# Port libomp contains:
+#   /opt/local/include/libomp/omp-tools.h
+#   /opt/local/include/libomp/omp.h
+#   /opt/local/include/libomp/ompt.h
+#   /opt/local/lib/libomp/libgomp.dylib
+#   /opt/local/lib/libomp/libiomp5.dylib
+#   /opt/local/lib/libomp/libomp.dylib
+#   /opt/local/share/doc/libomp/LICENSE.TXT
+#   /opt/local/share/doc/libomp/README.txt
+# """
```

Comment:
Can these examples be put in the docstring of the `compile_link_paths_from_manifest` function below? Something like (yes the double `::` is intended to make a block quote in sphinx, if this were to ever get documented):

```python

def compile_link_paths_from_manifest(cmd):
    """Parse include and library paths from OSX package managers.

    Example executions::

        # Homebrew
        $ brew ls --verbose libomp
        /opt/homebrew/Cellar/libomp/15.0.7/INSTALL_RECEIPT.json
        /opt/homebrew/Cellar/libomp/15.0.7/.brew/libomp.rb
        /opt/homebrew/Cellar/libomp/15.0.7/include/ompt.h
        /opt/homebrew/Cellar/libomp/15.0.7/include/omp.h
        /opt/homebrew/Cellar/libomp/15.0.7/include/omp-tools.h
        /opt/homebrew/Cellar/libomp/15.0.7/lib/libomp.dylib
        /opt/homebrew/Cellar/libomp/15.0.7/lib/libomp.a

        # MacPorts
        $ port contents libomp
        Port libomp contains:
          /opt/local/include/libomp/omp-tools.h
          /opt/local/include/libomp/omp.h
          /opt/local/include/libomp/ompt.h
          /opt/local/lib/libomp/libgomp.dylib
          /opt/local/lib/libomp/libiomp5.dylib
          /opt/local/lib/libomp/libomp.dylib
          /opt/local/share/doc/libomp/LICENSE.TXT
          /opt/local/share/doc/libomp/README.txt

    """
    from subprocess import run
```

Reply:
Included this in most recent push.